resources/install.rb

```diff
@@ -113,8 +114,10 @@
       mode '0644'
       source lazy { node.run_state['haproxy']['conf_template_source'][config_file] }
       cookbook lazy { node.run_state['haproxy']['conf_cookbook'][config_file] }
-      notifies :enable, 'poise_service[haproxy]', :immediately
-      notifies :restart, 'poise_service[haproxy]', :delayed
+      unless install_only
+        notifies :enable, 'poise_service[haproxy]', :immediately
```

Comment:
So what's the trick here for only installing that? 

Apologies for not reading that post, any chance of a summary of what we're achieving here by *not* restarting. 

Hopefully a bunch of people learn from it asa  result :)